### PR TITLE
adds markdown support to comment parsing

### PR DESCRIPTION
**Pull request by hkollmann**
fix issue https://github.com/qooxdoo/qxl.apiviewer/issues/26: HTML markup and code sections in source code docblocks not correctly converted.

### CODE REVIEW

File: test/test-jsdoc.js [hunk 1]
```diff
@@ -1,6 +1,93 @@
...
+* representing the remote resource is instantiated and actions are invoked on this resource.
+* A resource with its actions can be configured declaratively or programmatically.
+*
+* There is also {@link qx.io.rest.Resource} which uses {@link Resource} under the hood.
+* The main differences between them are:
+*
+* * The event object available in the listeners (e.g. @success()@, @getSuccess()@ and @getError()@) is
+*   a native JavaScript object instead of a qooxdoo object ({@link qx.event.type.Rest}):
+* ** See {@link qx.io.rest.Resource} vs. {@link Resource}
+* ** @event.getId()@ => @event.id@
```
Comment (cboulanger) [inline test/test-jsdoc.js]: What does the encapsulation in `@` do?

File: test/test-jsdoc.js [hunk 2]
```diff
@@ -1,6 +1,93 @@
...
+	let text = `
+* *strong*
+* __emphasis__
+* @light@              -> <code>light</code> 
+* @light@  @light@            -> <code>light</code> <code>light</code>
+* {@link Resource}     -> link?
+* @ignore(qx.*)
+`  
+  var test = qx.tool.compiler.jsdoc.Parser.parseComment(text);
+  console.log(test["@description"][0].body);
```
Comment (cboulanger) [inline test/test-jsdoc.js]: This doesn't really test anything, does it? Maybe you could save the output in a file and use the content as the "expected" value?

File: test/test-jsdoc.js (hunk 1 above)
Comment (hkollmann) [reply test/test-jsdoc.js]: @xxx@ found in some __index__.js covers the text into a <code>xxx</section>.

File: test/test-jsdoc.js (hunk 2 above)
Comment (hkollmann) [reply test/test-jsdoc.js]: Next time

File: test/test-jsdoc.js (hunk 1 above)
Comment (cboulanger) [reply test/test-jsdoc.js]: Ok, like backticks in MarkDown or `<code>...</code>`? We should replace this eventually, but that's beyond the scope of your PR.

Comment (hkollmann) [reply test/test-jsdoc.js]: You can use backticks now. It's an old - undocumented - qooxdoo feature.
Some others - like __xx__ are documented.
Just found it in your failure examples. 

http://www.qooxdoo.org/current/api/#qx.bom.rest :  The higlighted parts.

**Pull request merged**